src/ODBC/OdbcConnection.php

```diff
@@ -100,6 +101,14 @@ public function quoteIdentifier(string $str): string
         return '`' . str_replace('`', '``', $str) . '`';
     }
 
+    /**
+     * @param resource $stmt
+     */
+    protected function getQueryMetadata(string $query, $stmt): QueryMetadata
```

Comment:
Pri custom query zapisujeme do CSV header (pri ostatnych queries do manifestu).
Robili sme to tak, ze pri spracovani prveho riadku, sme zobrali kluce pola a to bola `$header`, ktora sa zapisala do CSV.

Lenze ak chceme vytvorit CSV aj v pripade, ze nie su ziadne riadky, ... tak tento postup nefunguje, lebo nemame ziadny prvy riadok. Zoznam stlpcov teda musime ziskat inak.

`Pdo/Odbc Connection` trieda dostala novu metodu, ktora vracia metadata pre query. Tieto metadata su potom sucastou `QueryResult`.
Pre PDO aj ODBC existuju funkcie, ktore nam tento zoznam stlpcov vratia, aj ked query nevratila ziadne riadky.

Pre ine DB technologie sa to moze implementovat aj inak (napr. cez SQL  `EXPLAIN/DESCRIBE` ...)